tutorials/xr/xr_action_map.rst

```diff
@@ -385,3 +385,124 @@ This is why many XR runtimes only use it as a last resort and will attempt to us
   This is our advice as well: limit your action map to the interaction profiles for devices you have actually tested your game with.
   The Oculus Touch controller is widely used as a fallback controller by many runtimes.
   If you are able to test your game using a Meta Rift or Quest and add this profile there is a high probability your game will work with other headsets.
+
+Binding Modifiers
+-----------------
+
+While one of the main goals of the action map is to remove the need for the application to know the hardware used, sometimes the hardware has physical differences that require inputs to be altered in ways other than how they are bound to actions. This need ranges from setting thresholds, to altering the inputs available on a controller.
+
+Currently OpenXR supports two binding modifiers that have been implemented in Godot but more are likely to follow in the future.
```

Comment:
> Currently OpenXR supports two binding modifiers that have been implemented in Godot **but more are likely to follow in the future.**

I'd suggest removing that part; its generally not super helpful to users *now*, and also looks bad if we don't end up following it up in the (near) future. We should IMO try to document the current state, not possible futures?

Reply:
Yes. In general the docs should:
- Document current state, written in present tense, in a way that will still read well in the future
- Avoid using the word "currently", though! It's usually rendundant
- Avoid mentioning future plans like this

Source: I looked through the docs recently and found a lot of instances where we didn't follow these guidelines and things needed to be corrected

Reply:
Agreed, removing this.